scss/index.scss

```diff
@@ -102,6 +102,13 @@
     @if nth($widgetInstance, 2) == 'rss' {
       @include rss(nth($widgetInstance, 3), nth($widgetInstance, 1), nth($widgetInstance, 4));
     }
+    @if nth($widgetInstance, 2) == 'lfd' {
```

Comment:
Approved but needs to be tested as I do not know if the solution I proposed works.